tests/e2e/bats/check_for_stackrox_OOMs.bats

```diff
@@ -11,16 +11,16 @@ load "../../../scripts/test_helpers.bats"
     source "${BATS_TEST_DIRNAME}/../lib.sh"
 
     run check_for_stackrox_OOMs "${BATS_TEST_TMPDIR}/oom-test"
-
-    with_oomkilled_test="$(ls ${ARTIFACT_DIR}/junit-misc/junit-OOMCheck-central-84bf956f94-bg6hr.xml)"
-    assert [ -f "$with_oomkilled_test" ]
-    run grep -q "was OOMKilled" "$with_oomkilled_test"
     assert_success
 
-    without_oomkilled_test="$(ls ${ARTIFACT_DIR}/junit-misc/junit-OOMCheck-sensor-67d98c67bf-v688m.xml)"
-    assert [ -f "$without_oomkilled_test" ]
-    run grep -q "was not OOMKilled" "$without_oomkilled_test"
+    run cat "${ARTIFACT_DIR}/junit-misc/junit-OOM Check.xml"
     assert_success
 
-    refute [ -f "${ARTIFACT_DIR}/junit-.xml" ]
```

Comment:
fwiw, this refute was added because of a bug in development. Might be worth keeping